django_freeradius/base/models.py

```diff
@@ -652,3 +656,77 @@ def expire(self):
         for u in users:
             u.is_active = False
             u.save()
+
+
+@python_2_unicode_compatible
+class AbstractRadiusProfile(TimeStampedEditableModel):
+    name = models.CharField(verbose_name=_('name'),
+                            max_length=128,
+                            help_text=_('A unique profile name'),
+                            db_index=True)
+    daily_session_limit = models.BigIntegerField(verbose_name=_('daily session limit'),
+                                                 blank=True,
+                                                 null=True)
+    daily_bandwidth_limit = models.BigIntegerField(verbose_name=_('daily bandwidth limit'),
+                                                   blank=True,
+                                                   null=True)
+    max_all_time_limit = models.BigIntegerField(verbose_name=('maximum all time session limit'),
+                                                blank=True,
+                                                null=True)
+    default = models.BooleanField(verbose_name=_('Use this profile as the default profile'),
+                                  default=False)
+
+    class Meta:
+        verbose_name = _('radius session and bandwidth limit')
+        verbose_name_plural = _('radius session and bandwidth limits')
+        abstract = True
+
+    def __str__(self):
+        return self.name
+
+    def save(self):
+        if self.default:
+            RadiusProfile = swapper.load_model('django_freeradius', 'RadiusProfile')
+            for rl in RadiusProfile.objects.filter(default=True):
```

Comment:
I think you can optimize this line, should be (writing by memory):

```python
RadiusProfile.objects.filter(default=True).update(default=False)
```

This will generate a single update query (but won't call the `save()` method of the model, which should be ok in this case.